scripts/cc-cli/src/utils/validate.ts

```diff
@@ -39,6 +39,5 @@ export async function chill(seed: string, api: ApiPromise) {
 
   const hash = await chillTx.signAndSend(account);
 
-  console.log(`Chill transaction sent with hash: ${hash.toHex()}`);
```

Comment:
Q: If that's the duplicate where is the other `console.log` call which produces this message? I haven't been able to find it in the code.

Reply:
It was in the `chill` function and in the `chillAction` which calls it.